go.mod

```diff
@@ -3,16 +3,27 @@ module github.com/scylladb/gocqlx/v2
 go 1.17
 
 require (
-	github.com/gocql/gocql v0.0.0-20211015133455-b225f9b53fa1
+
+	github.com/gocql/gocql v1.0.0
 	github.com/google/go-cmp v0.5.4
+	github.com/maraino/go-mock v0.0.0-20180321183845-4c74c434cd3a
 	github.com/psanford/memfs v0.0.0-20210214183328-a001468d78ef
 	github.com/scylladb/go-reflectx v1.0.1
-	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
+	github.com/stretchr/testify v1.7.1
```

Comment:
Can you try to get rid of this extra dependency `github.com/stretchr/testify`?

Reply:
The same here, used assertion on unit tests

Reply:
Done